python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -52,6 +52,10 @@ def lazyLoadJobApplication(this_job):
 
 def lazyLoadJobObject(this_job, this_attr):
 
+    if stripProxy(this_job)._getRegistry():
+        if stripProxy(this_job)._getRegistry().has_loaded(stripProxy(this_job)):
+            return getattr(this_job, this_attr)
```

Comment:
but my intention is that this may be called from the proxy layer in the future and for testing. I don't see anything intrinsically bad about this?